app/models/appliances.rb

```diff
@@ -1,7 +1,7 @@
 class Appliances < Base
   def list
     read_appdb_data.collect do |service|
-      service['appliance']
+      service['appliances']
```

Comment:
What happens when `service` is `nil` or when `service['appliances']` doesn't exist? Please, add at least `next unless service` and `.compact` the resulting array. !resolve